modin/pandas/indexing.py

```diff
@@ -267,7 +268,7 @@ def _compute_ndim(row_loc, col_loc):
     return ndim
 
 
-class _LocationIndexerBase(object):
+class _LocationIndexerBase(object, metaclass=LoggerMetaClass):
```

Comment:
Again, this is more internal - should it be logging with the `PANDAS_API` tag?

Reply:
Addressed in previous comment.